src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -252,7 +252,12 @@ void c_typecheck_baset::typecheck_expr_main(exprt &expr)
   }
   else if(expr.id()==ID_clang_builtin_convertvector)
   {
+    // This has one operand and a type, and acts like a typecast
+    DATA_INVARIANT(expr.operands().size()==1, "");
```

Comment:
Nit picking: spacing around `==`, `=`. Not so nit-picking: please move the comment into the second operand of `DATA_INVARIANT` (instead of the empty string).

Reply:
Fixed